appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap

```diff
@@ -3535,6 +3535,92 @@ exports[`appellant-case GET /appellant-case/valid/confirmation should render the
 </main>"
 `;
 
+exports[`appellant-case GET /appellant-case/valid/date should render the 500 error page if required data is not present in the session 1`] = `
+"<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full"><span class="govuk-caption-xl govuk-!-margin-bottom-1">Appeal undefined</span>
```

Comment:
It doesn't have the data (appeal undefined) if this shouldn't be a 500 error page